packages/expo-video/ios/VideoView.swift

```diff
@@ -4,11 +4,14 @@ import AVKit
 import ExpoModulesCore
 
 public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
+  private var isInPictureInPicture = false
   lazy var playerViewController = AVPlayerViewController()
 
   var player: AVPlayer? {
     didSet {
       playerViewController.player = player
+      // Now playing is updated by the `NowPlayingManager`
+      playerViewController.updatesNowPlayingInfoCenter = false
```

Comment:
If it's always `false`, can we set this in the `init` where the `playerViewController` is initialized?